### ISSUE TITLE
Rename `labelOptions` to `tooltipOptions`

### ISSUE THREAD

**Issue by schloerke** [rstudio/leaflet#511]
With the adoption of Leaflet.label into `L.tooltip`, if we'd like to copy the names fully, we should rename labelOptions (function and argument usage) to tooltipOptions.

This will probably involve deprecation / message functionality within the package.  

Or (current implementation) we could leave as is and not change our api, but just point to the documentation of tooltip options from our labelOptions documentation. 
https://github.com/rstudio/leaflet/blob/bfe4b0d877bbe4a1846df628d426cd980146b2ef/R/layers.R#L548

**Comment by MatthiasHinz** [comment]
I would suggest to also add (mention) the 'sticky'-parameter within the label option documentation. In the previous version, it was the default that the tooltip of a polygon follows the mouse cursor. Now it must be activated via `sticky = TRUE`.

**Comment by schloerke** [comment]
@MatthiasHinz Was `sticky=TRUE` the default behavior before? I can not find it in the codes for Leaflet.label or leaflet.js.  If so, should it be the default behavior still? (So I can document properly)

**Comment by tim-salabim** [comment]
The behaviour of the tooltip following the mouse was the default behaviour in the leaflet R package before. I don not know whether it also was in leafletjs. See also my comment [here](https://github.com/rstudio/leaflet/pull/492#issuecomment-365718445) - bullet 3. In my opinion it make more sense for the tooltip to be displayed at the mouse cursor than at some location representing the "center" of a feature.

**Comment by schloerke** [comment]
I completely agree with following the mouse vs "center" for a default.  The jarring feeling I get when it doesn't show up at my mouse is worth going against the default setting.

@tim-salabim Should the `sticky=TRUE` be added to labelOptions as a default option? This would effectively keep it as it was in prior version

**Comment by tim-salabim** [comment]
Yes, i think that is a good idea.

**Comment by schloerke** [comment]
`sticky=TRUE` implemented in #520
